### PR TITLE
Add links for product ingested traces

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Add links to the external guide for reducing the ingestion volume + link to the ingestion mechanisms page

### Motivation

Missing links

### Preview

https://docs-staging.datadoghq.com/antoine.dussault/guide_add_links/tracing/guide/trace_ingestion_volume_control/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_ingestion/mechanisms.md [hunk 1]
```diff
@@ -192,12 +192,12 @@ In Agent version 7.33 and forward, you can disable the rare sampler in the Agent
...
 
-| Product    | `ingestion_reason`                    |
-|------------|-------------------------------------|
-| Synthetic Monitoring | `synthetics` and `synthetics-browser` |
-| Real User Monitoring        | `RUM`                               |
-| Serverless | `lambda` and `xray`                   |
-| Application Security Monitoring     | `appsec`                            |
+| Product    | Ingestion Reason                    | Ingestion Mechanism Description |
+|------------|-------------------------------------|---------------------------------|
+| Synthetic Monitoring | `synthetics` and `synthetics-browser` | An HTTP or browser test generates a trace when the backend services are instrumented. [Find the backend trace][4] from the synthetic test run.
```
Comment (buraizu) [inline content/en/tracing/trace_ingestion/mechanisms.md:197]:
```suggestion
| Synthetic Monitoring | `synthetics` and `synthetics-browser` | An HTTP or browser test generates a trace when the backend services are instrumented. Find the backend trace from the [synthetic test][4] run.
```

File: content/en/tracing/trace_ingestion/mechanisms.md [hunk 2]
```diff
@@ -192,12 +192,12 @@ In Agent version 7.33 and forward, you can disable the rare sampler in the Agent
...
+| Real User Monitoring        | `RUM`                               | A browser request from a web or mobile application generates a trace when the backend services are instrumented. [Find the backend trace][5] from the RUM browser sessions and resources. |
```
Comment (buraizu) [inline content/en/tracing/trace_ingestion/mechanisms.md:198]:
```suggestion
| Real User Monitoring        | `RUM`                               | A browser request from a web or mobile application generates a trace when the backend services are instrumented. Find the backend trace from the [RUM browser sessions][5] and resources. |
```

File: content/en/tracing/trace_ingestion/mechanisms.md (hunk 1 above)
Comment (buraizu) [reply content/en/tracing/trace_ingestion/mechanisms.md:197]: Small update to make the link text more descriptive.

File: content/en/tracing/trace_ingestion/mechanisms.md (hunk 2 above)
Comment (buraizu) [reply content/en/tracing/trace_ingestion/mechanisms.md:198]: Small update to make the link text more descriptive.

File: content/en/tracing/trace_ingestion/mechanisms.md [hunk 3]
```diff
@@ -207,3 +207,7 @@ Some additional ingestion reasons are attributed to spans that are generated by
 [1]: /tracing/setup_overview/setup/
 [2]: /tracing/visualization/#trace-root-span
 [3]: /tracing/trace_ingestion/control_page
+[4]: /synthetics/apm/
+[5]: /real_user_monitoring/connect_rum_and_traces/
+[6]: /serverless/distributed_tracing/
```
Comment (buraizu) [inline content/en/tracing/trace_ingestion/mechanisms.md:212]:
```suggestion
[6]: /serverless/libraries_integrations/extension/
```

Comment (buraizu) [reply content/en/tracing/trace_ingestion/mechanisms.md:212]: Is this link intended to point to the Lambda extension, as the link text on line 199 would suggest? If the link is indeed meant for /serverless/distributed_tracing/, could you update the link text to "serverless applications" or something similar?

**Review by buraizu — CHANGES_REQUESTED** [review]
Small updates to clarify the link text

File: content/en/tracing/trace_ingestion/mechanisms.md [hunk 4]
```diff
@@ -192,12 +192,12 @@ In Agent version 7.33 and forward, you can disable the rare sampler in the Agent
...
-|------------|-------------------------------------|
-| Synthetic Monitoring | `synthetics` and `synthetics-browser` |
-| Real User Monitoring        | `RUM`                               |
-| Serverless | `lambda` and `xray`                   |
-| Application Security Monitoring     | `appsec`                            |
+| Product    | Ingestion Reason                    | Ingestion Mechanism Description |
+|------------|-------------------------------------|---------------------------------|
+| Synthetic Monitoring | `synthetics` and `synthetics-browser` | An HTTP or browser test generates a trace when the backend services are instrumented. [Find the backend trace][4] from the synthetic test run.
+| Real User Monitoring        | `RUM`                               | A browser request from a web or mobile application generates a trace when the backend services are instrumented. [Find the backend trace][5] from the RUM browser sessions and resources. |
+| Serverless | `lambda` and `xray`                   | Your traces received from the [Datadog Lambda Extension][6] or AWS X-Ray integration |
```
Comment (dussault-antoine) [inline content/en/tracing/trace_ingestion/mechanisms.md:199]:
```suggestion
| Serverless | `lambda` and `xray`                   | Your traces received from the [Serverless applications][6] traced with Datadog Tracing Libraries or the AWS X-Ray integration |
```

File: content/en/tracing/trace_ingestion/mechanisms.md [hunk 5]
```diff
@@ -192,12 +192,12 @@ In Agent version 7.33 and forward, you can disable the rare sampler in the Agent
...
-| Real User Monitoring        | `RUM`                               |
-| Serverless | `lambda` and `xray`                   |
-| Application Security Monitoring     | `appsec`                            |
+| Product    | Ingestion Reason                    | Ingestion Mechanism Description |
+|------------|-------------------------------------|---------------------------------|
+| Synthetic Monitoring | `synthetics` and `synthetics-browser` | An HTTP or browser test generates a trace when the backend services are instrumented. Find the backend trace from the [synthetic test][4] run.
+| Real User Monitoring        | `RUM`                               | A browser request from a web or mobile application generates a trace when the backend services are instrumented. Find the backend trace from the [RUM browser sessions][5] and resources. |
+| Serverless | `lambda` and `xray`                   | Your traces received from the [Serverless applications][6] traced with Datadog Tracing Libraries or the AWS X-Ray integration |
+| Application Security Monitoring     | `appsec`                            | Traces ingested from Datadog tracing libraries and flagged by [ASM][7] as a threat |
```
Comment (dussault-antoine) [inline content/en/tracing/trace_ingestion/mechanisms.md:200]:
```suggestion
| Serverless | `lambda` and `xray`                   | Your traces received from the [Serverless applications][6] traced with Datadog Tracing Libraries or the AWS X-Ray integration. |
| Application Security Monitoring     | `appsec`                            | Traces ingested from Datadog tracing libraries and flagged by [ASM][7] as a threat. |
```

**Review by buraizu — APPROVED** [review]

**Pull request merged**